exercises/practice/sublist/impl.mips

```diff
@@ -0,0 +1,19 @@
+# | Register | Usage     | Type    | Description                                                |
+# | -------- | --------- | ------- | ---------------------------------------------------------- |
+# | `$a0`    | input     | address | array one elements                                         |
+# | `$a1`    | input     | integer | size of array one, in words                                |
+# | `$a2`    | input     | address | array two elements                                         |
+# | `$a3`    | input     | integer | size of array two, in words                                |
+# | `$v0`    | output    | integer | `0` = equal, `1` = unequal, `2` = sublist, `3` = superlist |
+# | `$t0-9`  | temporary | any     | used for temporary storage                                 |
+
+.eqv UNEQUAL   0
+.eqv EQUAL     1
+.eqv SUBLIST   2
+.eqv SUPERLIST 3
```

Comment:
Did you purposefully leave this in? I don't mind it, but I don't think I've seen it done in other exercise. Same for the `li $v0, UNEQUAL` line below, where there's usually just the `jr $ra`.

Reply:
Yes, for a combination of reasons:

- We currently don't have any syllabus / learning exercises that tell people about `.eqv`.  One of the newly added resources mentions `.eqv`, but none of the other resources do. The `li $v0, UNEQUAL` has tutorial value.
- it helps document the expected return values.
- this approach is consistent with other tracks (C) using an enum
- the upcoming yacht exercise has a `category` input value. For MIPS this should be passed in as an int, not a string.  We can define `.eqv` values there too.